vaadin-context-menu-flow-parent/vaadin-context-menu-flow-integration-tests/src/test/java/com/vaadin/flow/component/contextmenu/it/SubMenuHelpersIT.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2000-2023 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.contextmenu.it;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.vaadin.flow.component.button.testbench.ButtonElement;
+import com.vaadin.flow.component.contextmenu.testbench.ContextMenuElement;
+import com.vaadin.flow.component.contextmenu.testbench.ContextMenuItemElement;
+import com.vaadin.flow.component.contextmenu.testbench.ContextMenuOverlayElement;
+import com.vaadin.flow.component.notification.testbench.NotificationElement;
+
+import com.vaadin.tests.AbstractComponentIT;
+import com.vaadin.flow.testutil.TestPath;
+
+@TestPath("vaadin-context-menu/sub-menu-helpers-test")
+public class SubMenuHelpersIT extends AbstractComponentIT {
```

Comment:
I think that it's not required to have SubMenuHelpersIT and the related `SubMenuPage`.
Can you please move `contextMenuSubMenuTest` to SubMenuIT -> the SubMenuPage already has the infrastructure to perform such test.

Then please move the `contextMenuOpenCloseTest` to the `ContextMenuPageIT`, or maybe even better, use the `$(ContextMenuElement.class)` and `isOpen()` in some of the existing tests there...

The point is to reduce the number of tests here - context menu opening, submenu opening and clicking are already tested in other tests. It makes sense to me to use the new TestBench Api in the existing tests to verify it's function (and usefullness).